<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DatabaseSettingsForm</class>
 <widget class="QDialog" name="DatabaseSettingsForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>30</y>
     <width>191</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Database Settings</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Database Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Database Description</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Default User Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_dbName">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>120</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_description">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>160</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_userName">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>200</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox_Response">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>260</y>
     <width>156</width>
     <height>23</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
